app/src/main/java/io/neurolab/main/MainActivity.java

```diff
@@ -35,20 +35,21 @@
 public class MainActivity extends AppCompatActivity
```

Comment:
Let's rename `MainActivity` to something more sensible. May be `NeuroLab`

Reply:
Sure :+1: